specification/apimanagement/developer-services-api/authorized/resources/confirmations.tsp

```diff
@@ -0,0 +1,28 @@
+import "@typespec/rest";
+import "../../models/common.tsp";
+import "../../models/confirmation.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.Core.Foundations;
+using DeveloperServicesApi.Models;
+
+namespace DeveloperServicesAuthorizedApi.Resources.Confirmations;
+
+interface Confirmations {
+  @route("/confirmations/users/{userId}/identities/basic")
+  @doc("Basic identity (Email) confirmation for a User.")
+  @post
+  CreateUserBasicIdentityConfirmation(
```

Comment:
This should be implemented with an operation template from Azure Core, either `ResourceAction` or `RpcOperation`.